pkgs/development/interpreters/python/cpython/default.nix

```diff
@@ -333,14 +333,11 @@ in with passthru; stdenv.mkDerivation {
     # Never even try to use lchmod on linux,
     # don't rely on detecting glibc-isms.
     "ac_cv_func_lchmod=no"
-  ] ++ optionals (libxcrypt != null) [
-    "CFLAGS=-I${libxcrypt}/include"
-    "LIBS=-L${libxcrypt}/lib"
```

Comment:
is this an opportunistic clean up or an accident?

Reply:
Passed libxcrypt via buildInputs, because it wouldn't be found. So very intentional,